mos-platform/nes/nesdoug/vram_buffer.s

```diff
@@ -6,8 +6,9 @@
 .section .noinit.vram_buf,"aw",@nobits
 .globl VRAM_BUF
 .balign 128
+.weak
```

Comment:
This doesn't do anything; the syntax is `.weak <names>`. So, this should be `.weak VRAM_BUF`.

Reply:
Oh hm... apparently `.weak` does the job of `.globl` too. Fixed.